test-app/tests/integration/components/headless-form-basic-test.gts

```diff
@@ -25,13 +25,19 @@ module('Integration Component HeadlessForm > Basics', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders form markup', async function (assert) {
-    await render(<template><HeadlessForm class="foo" novalidate /></template>);
+    await render(<template>
+      <HeadlessForm class="foo" autocomplete="off" />
+    </template>);
 
     assert.dom('form').exists('it renders as <form>');
     assert
       .dom('form')
       .hasClass('foo', 'it accepts custom HTML classes')
-      .hasAttribute('novalidate', '', 'it accepts arbitrary HTML attributes');
```

Comment:
Removed the example using `novalidate` here, because the form now has this _by default_. So testing for any other HTML attribute instead, to not have a possibly false positive here.